pom.xml

```diff
@@ -234,7 +242,21 @@
     <dependency>
       <groupId>org.jsoup</groupId>
       <artifactId>jsoup</artifactId>
-      <version>1.10.2</version>
+      <version>1.11.3</version>
+      <scope>test</scope>
```

Comment:
So this wasn't in the test scope before, what prompted you to put it there now?

Reply:
I've only changed the version, it was already on the test scope. I did so because JCasC uses this version.

Reply:
Ah, sorry, the diff was a bit weird :)

Reply:
yes, it was